src/Core/Services/ResolverMiddleware.cs

```diff
@@ -371,5 +387,19 @@ private static void SetNewMetadata(IMiddlewareContext context, IMetadata? metada
         {
             context.ScopedContextData = context.ScopedContextData.SetItem(_contextMetadata, metadata);
         }
+
+        /// <summary>
+        /// Track event when GraphQL request is received and send EntityName, Operation and Description for Telemetry.
+        /// </summary>
+        private static void TrackEventForGraphQLRequests(IMiddlewareContext context, TelemetryClient _telemetryClient)
+        {
+            _telemetryClient.TrackEvent("GraphQLRequestReceived", new Dictionary<string, string>
```

Comment:
Probably have constants for all the event names in that PR (I saw the comment from Sean to have these events in a separate PR)